addons/resource/static/tests/resource_test_helpers.js

```diff
@@ -1,13 +1,7 @@
 import { ResourceTask } from "./mock_server/mock_models/resource_task";
 import { ResourceResource } from "./mock_server/mock_models/resource_resource";
-import { mailModels } from "@mail/../tests/mail_test_helpers";
-import { defineModels } from "@web/../tests/web_test_helpers";
 
 export const resourceModels = {
     ResourceTask,
     ResourceResource,
 };
-
-export function defineResourceModels() {
-    return defineModels({ ...mailModels, ...resourceModels });
-}
```

Comment:
Keep the method, even if not used (especially in stable). And remove `mailModels` from it.